tools/python/transit/gtfs/download_gtfs.py

```diff
@@ -60,7 +60,7 @@ def get_feeds_links(data):
     gtfs_feeds_urls = []
 
     for feed in data:
-        if feed["spec"] != "gtfs":
+        if feed["spec"].lower() != "gtfs":
```

Comment:
Does it make sense to print a warning in this case? What are other possible values here?

Reply:
From what I can see:
- `MDS`
- `GBFS`
- `GTFS_RT`
- `GTFS`

I have no idea what is the difference

Reply:
Can you please add this list to the comments in this switch? These are different types of feeds.

Reply:
If there has been a switch from `gtfs` to `GTFS`, we should just change to `if feed["spec"] != "GTFS"`.
I have no ideas what the other `spec` means.